monkestation/code/modules/liquids/code/liquid_systems/liquid_groups.dm

```diff
@@ -221,6 +221,13 @@ GLOBAL_VAR_INIT(liquid_debug_colors, FALSE)
 			T.liquids.total_reagents += R.volume
 			alpha_setting += max(R.opacity * R.volume, 0.0001)
 			alpha_divisor += max(1 * R.volume, 0.0001)
+
+		//Ew but this is needed for edge cases in the new evaporation as its "technically" possible for numbers to balance out to 0 and cause division issues
```

Comment:
Couldn't this be fixed by adding a clamp with a minimum of 1 to the setting and divisor statements up there?

Reply:
no, as this proc handles multiple reagents, so you would end up with a lopsided opacity, in reality it doesn't matter as this on;y procs when something evaporates into null at the same tick its added to a liquid group so either way you don't see it. So in that sense yes you could